client/post-editor/editor-sidebar/index.jsx

```diff
@@ -27,8 +29,31 @@ export default class EditorSidebar extends Component {
 		isPrivate: PropTypes.bool,
 	}
 
+	constructor() {
+		super();
+		this.state = { showFeedback: false };
+	}
+
+	openFeedbackPane = () => {
+		this.setState( { showFeedback: true } );
+	};
+	closeFeedbackPane = () => {
+		this.setState( { showFeedback: false } );
+	};
```

Comment:
The main idea is to be conscious not to accidentally return something of meaning to the caller. The only concrete one I can think of is the DOM0 event handler example where `anchorElement.onclick = () => functionThatReturnsFalse()` prevents the default action when the link is clicked. It's usually not an issue, but I believe I've caused at least one bug in other situations by preferring the expression form of arrow function and incidentally returning a value.

This is now probably way too much discussion for a minor point. :) I like the expression form here and will update the lines.